src/aeb_fate_asm.erl

```diff
@@ -33,14 +48,15 @@
 %%%          false
 %%%       5. Strings
 %%%          "Hello"
-%%%       6. Empty map
+%%%       6. Map
 %%%          {}
+%%%          { 1 => { "foo" => true, "bar" => false}
 %%%       7. Lists
 %%%          []
 %%%          [1, 2]
 %%%       8. Bit field
 %%%          <000>
-%%%          <1010>
+%%%          <1010 1010>
```

Comment:
is this space allowed?

Reply:
Yes, though it might be nice to be able to group bit fields.